### PR TITLE
Add Russian localization

### PR DESCRIPTION
**Pull request by adipierro**

### CODE REVIEW

File: ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  Nudge
+
+  Created by Augustina DiPierro on 30/01/22.
+  
+*/
+
+//// Additional Device Information
+"Additional Device Information" = "Информация об устройстве";
+// Username
+"Username:" = "Username:";
```
Comment (erikng) [inline ru.lproj/Localizable.strings:12]: Did you forget to do this one?

**Review by erikng — CHANGES_REQUESTED** [review]

File: ru.lproj/Localizable.strings (hunk 1 above)
Comment (adipierro) [reply ru.lproj/Localizable.strings:12]: Hi. 'Username' in Russian might be too big for that window – 'Имя пользователя'. I considered using 'Логин', but it's slang'ish.

Comment (erikng) [reply ru.lproj/Localizable.strings]: If you'd like, push a change with the non slang version and I can cut a test release on this PR to try it.

Comment (adipierro) [reply ru.lproj/Localizable.strings]: Okay.

**Comment by erikng** [comment]
I built it yesterday https://github.com/macadmins/nudge/runs/5043590338?check_suite_focus=true

and sorry I'm very sick yesterday and today so will be slow to respond.

**Comment by erikng** [comment]
Screen Shot 2022-02-04 at 12 48 45 PM
Screen Shot 2022-02-04 at 12 48 55 PM

maybe fix `(Demo Mode)`?

**Pull request merged**